Document how to build all images on one machine with qemu-static (#6)
kinvolk/calico-hostendpoint-controller: README.md

ARCH=arm64 docker build -t kinvolk/calico-hostendpoint-controller:v0.0.2-arm64 --build-arg ARCH . && docker push kinvolk/calico-hostendpoint-controller:v0.0.2-arm64
```

> To build an image for another architecture which is not your native OS architecture there are multiple ways.

Reviewer: ```suggestion
There are multiple ways for building an image for another architecture which is not your native OS architecture.
```

> The most convenient one is to install the `qemu-user-static` package on your system, maybe followed by `sudo systemctl restart systemd-binfmt.service` to set up binary translation so that the Linux kernel can run binaries for other architectures.

Reviewer: Nit: I would break the long lines. Same for other long lines of course.

Author: done

> The most convenient one is to install the `qemu-user-static` package on your system, maybe followed by `sudo systemctl restart systemd-binfmt.service` to set up binary translation so that the Linux kernel can run binaries for other architectures.

Reviewer: ```suggestion
The most convenient one is to install the `qemu-user-static` package on your system to set up binary translation so that the Linux kernel can run binaries for other architectures.

>NOTE: You may have to run `sudo systemctl restart systemd-binfmt.service` after installing.
```

> Afterwards, the only temporary change needed is to explicitly use the Docker image for the other architecture instead of the multiarch `debian:9` image in the `FROM` line of the `Dockerfile`. For our case, you need to use `arm64v8/debian:9` because there is no `debian:9-arm64`.

Reviewer: ```suggestion
The downside of this approach is that a base image for the non-native architecture must be specified in the `FROM` directive of the Dockerfile instead of an image like `debian:9`, which is a multiarch image. For example, for building a Debian-based ARM image on an x86-64 machine you need to use `arm64v8/debian:9` (because there is no `debian:9-arm64`).
```

Author: done

> When done building, you can remove the temporary change – unfortunately the `FROM` directive does not handle build arguments to make this step even more convenient.

Reviewer: ```suggestion
When done building, you can change the Dockerfile back to the way it was before. Unfortunately the `FROM` directive does not handle build arguments.
```

Now make sure you have `"experimental": "enabled"` in your `~/.docker/config.json` (surrounded by `{` and `}` if the file is otherwise empty).

> When all images are build on the respective architectures and published they can be combined through a manifest to build a multiarch image:

Reviewer: ```suggestion
When all images are built on the respective architectures and pushed they can be combined through a manifest to build a multiarch image:
```
